zerver/views/user_settings.py

```diff
@@ -387,6 +388,11 @@ def json_change_settings(
     # Loop over user_profile.property_types
     request_settings = {k: v for k, v in locals().items() if k in user_profile.property_types}
     for k, v in request_settings.items():
+        if k == "default_language" and v is not None:
+            with override_language(v):
+                result["msg"] = _(
+                    "Saved. Please <a class='reload_link'>reload</a> for the change to take effect."
```

Comment:
I am not sure whether having `a` tags in text will make the translation work correctly. I know we handle it using `z-link` in frontend but not sure how to handle it here. Are there any existing examples where we do translation of text containing `a` element in backend?